flink-java/src/main/java/org/apache/flink/api/java/ExecutionEnvironment.java

```diff
@@ -826,6 +826,7 @@ public JobExecutionResult execute(String jobName) throws Exception {
 		try {
 			if (configuration.getBoolean(DeploymentOptions.ATTACHED)) {
 				lastJobExecutionResult = jobClient.getJobExecutionResult(userClassloader).get();
+				lastJobExecutionResult.printResult();
```

Comment:
I think we should not print this in the base execution environments. This prints something that cannot be turned off, and so far we only printed that when running from the CLI.

Now, this would print on `stdout` whenever a job is executed which could be considered as noise.

Reply:
Agree.